src/dmd/traits.d

```diff
@@ -671,6 +672,22 @@ Expression semanticTraits(TraitsExp e, Scope* sc)
 
         return isCopyable(t) ? True() : False();
     }
+    if (e.ident == Id.hasAliasing)
```

Comment:
Use the `== Id.` form in the `Rebindable` test.

Reply:
Ahh. Good catch.

Reply:
Thanks. Done.